2-race-in-cache/README.md

```diff
@@ -8,6 +8,8 @@ Also, try to get your solution down to less than 30 seconds to run tests.  *Hint
 
 *Note*: Map access is unsafe only when updates are occurring. As long as all goroutines are only reading and not changing the map, it is safe to access the map concurrently without synchronization. (See [https://golang.org/doc/faq#atomic_maps](https://golang.org/doc/faq#atomic_maps))
 
+Advanced Level: Get your solution down to less than 5 seconds for all tests.
+
```

Comment:
Would this be different depending on what kind of system the code runs? Like can people just run bad code on faster hardware to get it under 5 seconds?

Reply:
I don't think you can do that, because the mock db has a 20ms wait inside, so bad code on fast hardware would still wait for the whole 20ms, and if you run them sequentially 1500 times, that's 30 seconds.

Reply:
I'll think about how to minimize the hardware difference. Maybe by increasing the db wait time and reducing number of cycles